.github/workflows/canary.yml

```diff
@@ -17,7 +17,7 @@ jobs:
           node-version: '12.x'
           registry-url: 'https://registry.verdaccio.org'
       - name: Install
-        run: yarn install --frozen-lockfile --network-timeout 100000
+        run: yarn install
```

Comment:
`--frozen-lockfile` is now `--immutable` https://yarnpkg.com/cli/install#options

Reply:
thanks ! I did not see this one :)